### DISCUSSION TITLE
Why has the font changed (to the worse)?

### DISCUSSION THREAD

**Discussion by UnlimitedCookies** [mpv-player/mpv#14267] [General]
Heya, I'm on Archlinux with wayland and Intel integrated graphics. Since the last update it seems the font has changed (and I don't like it). Is this due to a dependency change?

**Comment by guidocella** [comment]
https://github.com/libass/libass/issues/770

**Comment by UnlimitedCookies** [comment]
Yay.
Downgrade to avoid the regression:
`sudo pacman -U https://archive.archlinux.org/packages/l/libass/libass-0.17.1-4-x86_64.pkg.tar.zst`
